lib/ember-test-helpers/test-module.js

```diff
@@ -4,6 +4,13 @@ import { Klass } from 'klassy';
 import { getResolver } from './test-resolver';
 import buildRegistry from './build-registry';
 
+function hasEmberVersion(major, minor) {
```

Comment:
Can you move this to its own file?  We have a scenario in tests (for closure actions) where we are using a regexp to determine if a given test should run, and this is a much cleaner solution.

Something like `/lib/ember-test-helpers/matches-ember-version.js`?